app/viewmodels/journeys/taskList/TradeJourneyStatusesViewModel.scala

```diff
@@ -64,9 +68,10 @@ object TradeJourneyStatusesViewModel {
     val status: JourneyStatus = getJourneyStatus(journey, dependentJourneyIsFinishedForClickableLink)(journeyStatuses.journeyStatuses)
     val keyString             = messages(s"journeys.$journey")
     val href = journey match {
-      case Abroad => getAbroadUrl(status, businessId, taxYear)
-      case Income => getIncomeUrl(status, businessId, taxYear)
-      case _      => "#"
+      case Abroad      => getAbroadUrl(status, businessId, taxYear)
+      case Income      => getIncomeUrl(status, businessId, taxYear)
+      case Adjustments => "#"
+      case _           => "#"
```

Comment:
I'd make a 'getAdjustmentsUrl(status, businessId, taxYear)' method at the bottom like the others and put the "#" in the method

Reply:
Done :)